src/test/scala/com/redislabs/provider/redis/rdd/RedisSqlStandaloneSuite.scala

```diff
@@ -20,7 +21,10 @@ class RedisSqlStandaloneSuite extends FunSuite with ENV with BeforeAndAfterAll w
     spark = SparkSession.builder().config(conf).getOrCreate()
   }
 
-  case class Person(name: String, age: Int, address: String, salary: Double)
+  private val data = Seq(
```

Comment:
Since you made it shared between tests, should we remove `val data = ...` on line 34 ?